time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -108,10 +108,20 @@
     "uri",
 ])
 
+# custom attributes filter
+attributes_filter.add_argument('month', required=False,
+                                    store_missing=False,
+                                    help="(Filter) month to filter",
```

Comment:
LoL, lets better the filter prefix, which I added to the description of fields to make it clearer that it is a description of a filter. Does that make sense?
I would leave a help message like:
>Month to filter by